src/contracts/examples/fastlane-online/SolverGateway.sol

```diff
@@ -219,17 +221,31 @@ contract SolverGateway is OuterHelpers {
         );
 
         // Check can be grokked more easily in the following format:
-        //      solverOpScore    _cumulativeScore (unweighted)
-        // if  -------------- >  ------------------------------  * 2
-        //      solverOpGas             totalGas
-
-        if (_score * userOp.gas > _cumulativeScore * solverOp.gas * 2) {
-            if (_cumulativeGasReserved + USER_GAS_BUFFER + (solverOp.gas * 2) < userOp.gas) {
+        //      solverOpScore     _cumulativeScore (unweighted)
+        // if  -------------- >=  ------------------------------
+        //      solverOpGas          _cumulativeGasReserved
+
+        console.log("Score comp:");
```

Comment:
Leftover Debug statements?

Reply:
Yeah, good catch, will remove before merging this, find them helpful to sense check the math so left them in for reviewers for now

Reply:
Removed them here https://github.com/FastLane-Labs/atlas/pull/410/commits/820fcdbf85978acae0d31b85659747156e5cf531